assets/js/plugin-check-admin.js

```diff
@@ -252,9 +252,10 @@
 	 */
 	function renderResults( results ) {
 		const { errors, warnings } = results;
-
+		let errorWarningMessage = false;
```

Comment:
@mukeshpanchal27 could we not check the `errors` or `warnings` length here to see if there are errors? Instead of always updating the variable in for loops?

Reply:
On second thought, I think we'd also want to move this logic out to the `runChecks`. This `renderResults` function is run for each individual check, so when more than one check is run the complete message will display multiple times.